### ISSUE TITLE
Exampl app - Errors thrown when building on Xcode 13.3.1

### ISSUE THREAD

**Issue by bielikb** [krzysztofzablocki/AutomaticSettings#8]
Steps to reproduce:
1. Run Example App
2. Errors thrown
<img width="849" alt="Screenshot 2022-04-28 at 11 28 30" src="https://user-images.githubusercontent.com/4924709/165722537-f6481bbd-3aa4-4585-82dc-e3fbdc65dbc2.png">

**Comment by psi-gh** [comment]
Yeah, the first error appeared in "template project' of SwiftyStack when you compile. AutomaticSettings version was set to 1.1.0
Setting it to v1.2.3 fixed the problem.
